.travis.yml

```diff
@@ -149,6 +149,7 @@ jobs:
         INSTALL_AIRFLOW_VERSION="1.10.6"
       stage: test
       script: ./scripts/ci/ci_prepare_and_test_backport_packages.sh
+install: skip
```

Comment:
sry, what are we skipping here?

Reply:
Sorry that was quick. I had no time for explanation. I realized our build started to take long time.... and it turned out Travis is trying to be smart and when it sees requirements.txt it tries to install it automatically in the host...

That's what we are skipping :).

Reply:
got it

Reply:
:man_facepalming: